frontend/components/deck/DeckOverview.tsx

```diff
@@ -0,0 +1,45 @@
+import Deck from "./Deck";
+
+interface DeckOverviewProps {
+	/**
+	 * unique identifier
+	 */
+	id: string;
+	/**
+	 * Group name
+	 */
+	name: string;
+	/**
+	 * Decks to display in a group
+	 */
+	decks: { name: string; count: number }[];
+	/**
+	 * Additional classes
+	 */
+	className?: string;
+}
+
+/**
+ * UI component for displaying a group of decks
+ */
+export default function DeckOverview({
+	id,
+	name,
+	decks,
+	className,
+}: DeckOverviewProps) {
+	return (
+		<div
+			id={id}
+			className={`flex flex-col space-y-2 rounded-md ${className ?? ""}`}
+		>
+			<div className="text-lg font-bold text-darkblue">{name}</div>
+			<div className="flex snap-x flex-row space-x-5 overflow-x-scroll">
+				{decks.map((deck, i) => (
+					<Deck id="DeckId" key={i} deck={deck} />
```

Comment:
Even though it's just mock data, the `<div id="">` should still be unique. May consider adding at least the index for unique ids.

```suggestion
					<Deck id=`DeckId_{i}` key={i} deck={deck} />
```